src/Catty/IO/ProjectManager.swift

```diff
@@ -75,6 +75,13 @@
                                          thumbnailFrameSize: CGSize(width: CGFloat(kPreviewThumbnailWidth),
                                                                     height: CGFloat(kPreviewThumbnailHeight)))
 
+        let migrator = ProjectMigrator(fileManager: fileManager)
```

Comment:
Please do not use the migrator when creating an empty new project. Instead, please create the clean project with all directories here. Please add or extend some tests for this method. PS: you do not have to move any screenshots here.